src/data/learnQuizzes/index.ts

```diff
@@ -0,0 +1,32 @@
+// Import data types
+import { RawQuizzes } from "../../types"
+
+// Declare hash-map of quizzes based on slug key
+const quizzes: RawQuizzes = {
+  "what-is-ethereum": {
+    title: "What is Ethereum?",
+    questions: ["a001", "a002", "a003", "a004", "a005"],
```

Comment:
Why are these needed? Should we make these more intelligible to humans?

Reply:
The benefit here (separating individual questions from quizzes) is so we have the option to use any particular question in multiple quizzes if desired. ie, we currently reuse the "Who should you give your seed phrase / private keys to?" question in both the "wallets" and "security" quizzes.

Using an ID also helps reduce issues with tracking if we ever need to adjust the copy of a question. I had originally left a TODO to auto-generate an ID using a hash of the prompt (or something along those lines), but for the reason just mentioned I think it may be best to just add a manual ID. The number of questions should be within a reasonable amount where this isn't going to add significant overhead imo.